shapely/tests/geometry/test_point.py

```diff
@@ -89,7 +89,7 @@ def test_from_invalid():
         Point(1, 2, 3, 4)
 
     # this worked in shapely 1.x, just ignoring the other coords
-    with pytest.raises(ValueError, match="Invalid values passed"):
+    with pytest.raises(TypeError, match="only size-1 arrays"):
```

Comment:
The current error is not super informative, but this is also not an improvement (and also ValueError is better I think). I'll add a comment on the actual code about maybe catching this explicitly and raising a better error message.